CHANGELOG.md

```diff
@@ -6,30 +6,61 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
-If you use compression with a non-default collation on a segmentby-column you might have to recompress the affected hypertable.
+## 2.7.0 (2022-05-24)
+
+This release adds major new features since the 2.6.1 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+
+* Optimize continuous aggregate query performance and storage
+* The following query clauses and functions can now be used in a continuous
+  aggregate: FILTER, DISTINCT, ORDER BY as well as [Ordered-Set Aggregate](https://www.postgresql.org/docs/current/functions-aggregate.html#FUNCTIONS-ORDEREDSET-TABLE)
+  and [Hypothetical-Set Aggregate](https://www.postgresql.org/docs/current/functions-aggregate.html#FUNCTIONS-HYPOTHETICAL-TABLE)
+* Optimize now() query planning time
+* Improve COPY insert performance
+* Improve performance of UPDATE/DELETE on PG14 by excluding chunks
+
+This release also includes several bug fixes.
+
+If you are upgrading from a previous version and were using compression
```

Comment:
@svenklemm what's the consequence of not following this advice? Does it give an error or incorrect result, or something else?

Reply:
If you order by this column, the order can be incorrect (it would use a default collation, not the one you specified).

Reply:
But that affect query results? E.g., would the result be incorrect or is it just less efficient?

I am asking, because I think we need provide context when we write these things. For example:

```
After upgrading, it is recommended that compressed hypertables are recompressed if they use a non-default collection. Otherwise queries might be slower than expected.
```